File: src/leetcode/p1769/java/Solution.java
package leetcode.p1769.java;

import java.util.ArrayList;
import java.util.List;

public class Solution {
    public int[] minOperations(String boxes) {
        int[] result = new int[boxes.length()];
        String[] box = boxes.split("");
        List<Integer> list = new ArrayList();

        for (int i = 0; i < box.length; i++) {
            if (box[i].equals("1")) {
                list.add(i);
            }
        }
        for (int i = 0; i < result.length; i++) {
            for (int j = 0; j < list.size(); j++){
                System.out.println(Math.abs(i - list.get(j)));
                result[i] +=  Math.abs(i - list.get(j));
            }
        }
        return result;
    }
}

File: src/leetcode/p1769/kotlin/Solution.kt
package leetcode.p1769.kotlin

import kotlin.math.abs

class Solution {
    fun minOperations(boxes: String): IntArray {
        val list = mutableListOf<Int>()
        val resultArray = IntArray(boxes.length)

        for (box in boxes.indices) {
            if (boxes[box] == '1') {
                list.add(box);
            }
        }
        for (box in boxes.indices) {
            for (l in list) {
                println(abs(box - l))
                resultArray[box] += abs(box - l)
            }
        }
        return resultArray
    }
}